platform-emf/src/main/java/com/softicar/platform/emf/management/importing/variable/find/EmfImportVariablesFinder.java

```diff
@@ -6,7 +6,7 @@
 
 public class EmfImportVariablesFinder {
 
-	private static final String VARIABLE_PATTERN = "\\$\\$[a-z]\\$\\$";
+	private static final String VARIABLE_PATTERN = "\\$\\$[a-z]+\\$\\$";
```

Comment:
I think we should allow `[a-zA-Z0-9_]` ... and probably even spaces.
Being less restrictive here will result in improved legibility of multi-word variable names. (e.g. `businessPartnerName` or `business_partner_name` instead of `businesspartnername`)

Reply:
See new ACCs of story